Sources/Shared/Notifications/NotificationCommands/NotificationsCommandManager.swift

```diff
@@ -94,6 +98,20 @@ private struct HandlerClearNotification: NotificationCommandHandler {
     }
 }
 
+@available(watchOS 9, *)
```

Comment:
Let's remove this available check and move it under the `#if os(iOS)` on line 115